### PR TITLE
protobuf 3.11.0

### PR DESCRIPTION
**Pull request by AustinShalit**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/protobuf.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@ class Protobuf < Formula
   desc "Protocol buffers (Google's data interchange format)"
   homepage "https://github.com/protocolbuffers/protobuf/"
   url "https://github.com/protocolbuffers/protobuf.git",
-      :tag      => "v3.10.1",
-      :revision => "d09d649aea36f02c03f8396ba39a8d4db8a607e4"
+      :tag      => "3.11.0",
```
Comment (Bo98) [inline Formula/protobuf.rb]: This tag does not exist. Did you mean `v3.11.0`?

**Review by Bo98 — CHANGES_REQUESTED** [review]

File: Formula/protobuf.rb (hunk 1 above)
Comment (AustinShalit) [reply Formula/protobuf.rb]: Yes. I did. Fixed.

**Comment by AustinShalit** [comment]
retest this please

**Comment by EricFromCanada** [comment]
According to the [test output](https://jenkins.brew.sh/job/Homebrew%20Core%20Pull%20Requests/52611/version=highsierra/console), you need to run `brew bump-revision` for the following: `apache-arrow apache-arrow-glib caffe grpc libphonenumber libpulsar libtorch mosh protobuf-c shogun zbackup`

**Review by Bo98 — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]

**Pull request closed without merging**